sys/net/sock/sock_util.c

```diff
@@ -170,7 +170,7 @@ int _parse_netif(sock_udp_ep_t *ep_out, char *netifstart)
 {
     char *netifend;
     size_t netiflen;
-    char netifbuf[NETIF_STR_LEN + 1];
+    char netifbuf[NETIF_STR_LEN + 1] = {0};
```

Comment:
alternatively we could do `netifbuf[netiflen] = 0;`, which might be faster (slightly less code than this implicit memset call), but that would be quite context sensitive.

Reply:
I checked, on nrf52840dk, the codesize stays the same.

Reply:
which is weird. :) I checked the binaries with elf_diff, _parse_netif() grows by 8 bytes. maybe alignment luck makes the whole link stay the same.